### DISCUSSION TITLE
Audio Version?

### DISCUSSION THREAD

**Discussion by rl1234567890** [DivinumOfficium/divinum-officium#4142] [Ideas]
This is a long shot and I put it here only for reference. It would be nice to produce an audio version. The present logic of the program could be used to copy and paste mp3 files of psalms, collects, hymns, etc. into a single mp3 to download using  [mp3wrap](http://mp3wrap.sourceforge.net/). I have [experience in recording Latin](https://librivox.org/search?title=&author=&reader=bedwere&keywords=&genre_id=0&status=all&project_type=either&recorded_language=39&sort_order=catalog_date&search_page=1&search_form=advanced) and I would be happy to read everything.

**Comment by petermottola** [comment]
I, for one, would appreciate that functionality!  We may not need to make recordings, if we can find chants of the various psalms, e.g.:  [https://www.spreaker.com/user/monksofnorcia](https://www.spreaker.com/user/monksofnorcia).  
In future the sorting/assembling could be made to take into account different psalm tones (and settings of the _Te lucis_, etc.) based on rank of feast.
A read version would no doubt be preferred by some for the sake of brevity, but these are by no means mutually exclusive!

**Comment by lucarabistouille** [comment]
There are also the recordings by the abbey of **Le Barroux** (France): e.g. http://barrouxchant.com/#2016-12-19 (this abbey is the one who has accepted to give the right to DO to reproduce its translation of the missal and of the diurnal)…

We could also imagine to provide links to the gregorian scores. The gregobase has many of them: http://gregobase.selapa.net/scores.php
